src/coreComponents/unitTests/dataRepositoryTests/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+#
```

Comment:
Is the goal to eventually move the unit tests back into the components? I think pretty much all of these shouldn't depend on anything outside of `dataRepository`.

Reply:
That is the goal....every unit test should depend only on the component that it tests.